expo/expo
# [ios] Use stable manifest ID where applicable

# Why

Same as https://github.com/expo/expo/pull/12964 but for iOS.

# How

Same as https://github.com/expo/expo/pull/12964.

# Test Plan

- Run SDK 39, 40, 41 apps as a sanity check.
- Run a transferred app, ensure that `originalFullName` is used.
